dashboard/db/migrate/20200317042200_create_contact_rollups_final.rb

```diff
@@ -0,0 +1,9 @@
+class CreateContactRollupsFinal < ActiveRecord::Migration[5.0]
+  def change
+    create_table :contact_rollups_final do |t|
+      t.string :email, null: false, index: {unique: true}
```

Comment:
I had this question on my migration as well -- if we have a unique index on email, will that cause problems if we add a soft delete option? I think here we'd actually only want to hard delete (in the case of account deletion), but just wondering.

Reply:
We likely just need hard delete since we're rebuilding everything in each run. However, if we want to use soft delete, we can change the indexes following this instruction https://github.com/rubysherpas/paranoia#about-indexes.